packages/kit/src/runtime/app/forms.js

```diff
@@ -130,6 +130,11 @@ export function enhance(form_element, submit = () => {}) {
 
 	/** @param {SubmitEvent} event */
 	async function handle_submit(event) {
+		const method = event.submitter?.hasAttribute('formmethod')
+			? /** @type {HTMLButtonElement | HTMLInputElement} */ (event.submitter).formMethod
+			: clone(form_element).method;
+		if (method === 'dialog') return;
```

Comment:
```suggestion
		if (method !== 'post') return;
```

We should be able to simplify the if since that is the only method `use:enhance` supports